src/main/Config.h

```diff
@@ -308,8 +308,9 @@ class Config : public std::enable_shared_from_this<Config>
     void adjust();
 
     std::string toShortString(PublicKey const& pk) const;
-    std::string toStrKey(PublicKey const& pk, bool& isAlias) const;
+    std::string toStrKeyAuto(PublicKey const& pk, bool& isAlias) const;
```

Comment:
This isn't actually used by anything at the end of this PR. Is that intentional?